apps/judicial-system/web/src/utils/hooks/useIsMobile/useIsMobile.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react'
+
+const useIsMobile = () => {
+  const [isMobile, setIsMobile] = useState(false)
+
+  useEffect(() => {
```

Comment:
Do we have to use `useEffect` here?

Reply:
We use useEffect with an empty dependency array to ensure that the effect runs once when the component mounts, i.e. we only want to run the effect once when the component mounts.